src/content_script/mastodonInject.js

```diff
@@ -168,8 +169,7 @@ function initInjections() {
  */
 async function init() {
     if (typeof MASTODON_INJECTED_CLASS === "undefined"){
-        // eslint-disable-next-line vars-on-top, no-var
-        var MASTODON_INJECTED_CLASS = true;
+        window.MASTODON_INJECTED_CLASS = true;
```

Comment:
IIRC this was previously changed to `var`. What about [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) instead?